### ISSUE TITLE
pandas.read_hdf

### ISSUE THREAD

**Issue by alanchalk** [nielsborie/ml-docker#1]
Hi Niels,

Many thanks for this image which is very useful.
pandas.read_hdf requires the tables module.  Would you be able to add
pip install --upgrade tables
to your docker file?

(I have raised this with jupyter as sorting it in the earliest image that imports pandas probably makes most sense.  It might be that they prefer not to add it and this extra module fits quite well with the long list of optional extras at the end of your docker file...

Also, your image does not seem to have a tag (other than latest).  Are you able to give it a tag - so that if you change it we know which image we are pulling?

**Comment by nielsborie** [comment]
Hello Alan, 

Sorry for the late answer! I just had merged the pull request for the upgrade of "tables". 
I tested it locally with this script and it worked fine: 
```python
import numpy as np
import pandas as pd

df = pd.DataFrame({
    'A': [1, 2, 3, 4, 5, 1, 2, 3, 4, 5, 1, 2],
    'B': [0, 0, 1, 1, 1, 0, 1, 0, 1, 0, 0, 1],
    'C': [34, 32, 35, 34, 31, 34, 29, 34, 12, 34, 32, 34],
    'D': [11, 15, 22, 15, 9, 15, 11, 15, 14, 15, 13, 15]})

df.to_hdf('/tmp/out.h5', 'results_table', mode='w', data_columns=['A'],
          format='table')

print(pd.read_hdf('/tmp/out.h5', 'results_table', where='A in [1,3,4]'))
```

__Can you pull the latest image, try this script on your side and validate that this is the behavior you expected?__

---

* **About your docker tag request** :

I totally agree, it's a good idea to start tagging future versions of the image! ( Until now I hadn't felt the need because... I was the only one using this image ^^ )

As a result, **I just pushed the tag v.0.0** which corresponds to the version before the upgrade of "tables" (your correponded issue), and tag this new version **as latest**. 
So far, I've done it manually. But I think later this week I will push new things to this repository to add a minimal form of CI (Makefile, JenkinsFile ... )!

---

Besides, I noticed that you cloned this repo and made your own modifications: if you think there are features that could be useful, don't hesitate to fork and propose your own pull request ! ;)

Cheers

**Comment by alanchalk** [comment]
Hi Niels,

Thanks.

I’m teaching some students at a University and I had to get an image to them by start of this week - sorry! - otherwise I would have waited for your response.  At the time (last week) I was brand new to docker and I had no idea what to do and did not want to mess up anything in yours.

In my modifications, mainly I deleted quite a few of the extra packages that you load at the end - just to make the code that creates the image easier for me to read.  I also updated some packages (because the jupyter notebook on which it is based has an old version of one of them (pandas I think or sk-learn).  Finally I installed statsmods from master rather than pip because statsmods on pip does not run with the latest version of scripy (it is a known bug).  And actually I could not get tables to work!!  Each time I added it, the build failed!  

When I get a chance I will add any useful mods I did to your images and leave it as a pull request.  

Very many thanks for your everything.

Regards

Alan
